toolset/benchmark/framework_test.py

```diff
@@ -195,139 +196,147 @@ def tee_output(prefix, line):
+    except Exception as e:
       tee_output(prefix, "Docker build failed; terminating\n")
+      print(e)
       return 1
-        
+
 
     ##########################
     # Run the Docker container
     ##########################
-    p = subprocess.Popen(["docker", "run", "--rm", "-p", "%s:%s" % (self.port, self.port), "--network=host", "tfb/test/%s" % self.name],
-          stdout=subprocess.PIPE,
-          stderr=subprocess.STDOUT)
-    nbsr = setup_util.NonBlockingStreamReader(p.stdout,
-      "%s: framework processes have terminated" % self.name)
-
-    # Set a limit on total execution time of setup.sh
-    timeout = datetime.now() + timedelta(minutes = 105)
-    time_remaining = timeout - datetime.now()
-
-    # Need to print to stdout once every 10 minutes or Travis-CI will abort
-    travis_timeout = datetime.now() + timedelta(minutes = 5)
-
-    # Flush output until docker run work is finished. This is
-    # either a) when docker run exits b) when the port is bound
-    # c) when we run out of time. 
-    prefix = "Server %s: " % self.name
-    while (p.poll() is None
-      and not self.benchmarker.is_port_bound(self.port)
-      and not time_remaining.total_seconds() < 0):
-
-      # The conditions above are slow to check, so
-      # we will delay output substantially if we only
-      # print one line per condition check.
-      # Adding a tight loop here mitigates the effect,
-      # ensuring that most of the output directly from
-      # docker is sent to tee_output before the outer
-      # loop exits and prints things like "docker exited"
-      for i in xrange(10):
-        try:
-          line = nbsr.readline(0.05)
-          if line:
-            tee_output(prefix, line)
 
-            # Reset Travis-CI timer
-            travis_timeout = datetime.now() + timedelta(minutes = 5)
-        except setup_util.EndOfStream:
-          tee_output(prefix, "Docker has terminated\n")
-          break
-      time_remaining = timeout - datetime.now()
+    for test_docker_file in test_docker_files:
+        test_docker_file_name = test_docker_file.replace(".dockerfile", "")
+        test_docker_file_path = os.path.join(self.directory, "%s.dockerfile" % self.name)
+        p = subprocess.Popen(["docker", "run", "--rm", "--network=host", "tfb/test/%s" % test_docker_file_name],
+              stdout=subprocess.PIPE,
+              stderr=subprocess.STDOUT)
+        nbsr = setup_util.NonBlockingStreamReader(p.stdout,
+          "%s: framework processes have terminated" % self.name)
 
-      if (travis_timeout - datetime.now()).total_seconds() < 0:
-        sys.stdout.write(prefix + 'Printing so Travis-CI does not time out\n')
-        sys.stdout.write(prefix + "Status: Poll: %s, Port %s bound: %s, Time Left: %s\n" % (
-          p.poll(), self.port, self.benchmarker.is_port_bound(self.port), time_remaining))
-        sys.stdout.flush()
+        # Set a limit on total execution time of setup.sh
```

Comment:
There's no such thing as `setup.sh` anymore...

Reply:
this is git diffing being git diffing, i did not "add" that line